rectools/utils/config.py

```diff
@@ -1,4 +1,4 @@
-#  Copyright 2024 MTS (Mobile Telesystems)
+#  Copyright 2024-2025 MTS (Mobile Telesystems)
```

Comment:
We haven't touched it this year. Why was it changed?

Reply:
No idea

Reply:
1. Long time ago you created `experimental/sasrec` branch
2. In November I added this file without the license header
3. In the beginning of December you merged the `main` branch to this branch and brought this file
4. On the Dec 11 you made a release and added the license header to the `main` branch - so **the actual last change of the file was in 2024**
5. On Jan 20 you merged again the `main` branch to this branch, so **the latest commit in this branch was in 2025**


Let's do this:
1. Now just remove it manually
2. After we merge this branch into the `main`, we will run the script again. 
3. I'm not sure what will be the result. Hopefully, it will work correctly. If not, I'll try to modify the script

Reply:
Done (but we can just really keep it with changed year, this is no big deal)